### PR TITLE
Minor update bitcoin_ko_KR.ts

### PR DESCRIPTION
**Pull request by FultonDoge**
not a programmer, unable to test run the results of my translation
translated new? untranslated lines 1773, 1779, 2761, 4573, 4577, i dont believe there are any more untranslated lines
in the case of line 2761 reused identical line 2737 translation with minor update to replace satoshi with koinu for 2761 translation and both to make more sense.
assumed for line 4573 that %s and %i would print as %s and %i instead of being replaced by a string or integer

### CODE REVIEW

**Comment by slightlyskepticalpotat** [comment]
Yeah, as far as I know `%s` and `%i` are supposed to stay in translations everywhere. The "linter" check checks if your translations are syntactically correct and will fail if you miss a quote or anything.
image

**Comment by patricklodder** [comment]
Added help wanted. If someone that speaks Korean could give a quick glance, that'd be awesome.

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 1]
```diff
@@ -4571,11 +4571,11 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>Execute command when a wallet transaction changes (%s in cmd is replaced by TxID, %i with block height, with a value of 0 if tx is no longer in chaintip)</source>
-        <translation type="unfinished"></translation>
+        <translation>지갑 트랜잭션 변화시 명령 실행 (명령 안의 %s는 TxID, %i 는 블록높이로 변경되며 트랜잭션이 더 이상 chaintip 안이 아닌 경우에는 0으로 대체된다)</translation>
```
Comment (ilsubyeega) [inline src/qt/locale/bitcoin_ko_KR.ts:4574]:
```suggestion
        <translation>지갑 트랜잭션 변화시 명령 실행 (명령 안의 %s는 TxID, %i 는 블록높이로 변경되며 트랜잭션이 더 이상 chaintip 안이 아닌 경우에는 0으로 대체됩니다.)</translation>
```

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 2]
```diff
@@ -1771,13 +1771,13 @@ p, li { white-space: pre-wrap; }
         <source>BIP70 payment requests are deprecated and disabled by default. Restart with -enable-bip70 if you absolutely have to use this functionality.
 
 Use this functionality with extreme caution.</source>
-        <translation type="unfinished"></translation>
+        <translation>해당 기능 사용시 매우 주의하기 바랍니다</translation>
```
Comment (ilsubyeega) [inline src/qt/locale/bitcoin_ko_KR.ts:1774]: For me, `the sentence` in the source above doesn't look translated into `translation`.

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 3]
```diff
@@ -1771,13 +1771,13 @@ p, li { white-space: pre-wrap; }
...
     </message>
     <message>
         <source>Payment request file handling is disabled by default. Restart with -enable-bip70 if you absolutely have to use this functionality.
 
 Use this functionality with extreme caution.</source>
-        <translation type="unfinished"></translation>
+        <translation>해당 기능 사용시 매우 주의하기 바랍니다</translation>
```
Comment (ilsubyeega) [inline src/qt/locale/bitcoin_ko_KR.ts:1780]: Other this ^, `the sentence` in the source above doesn't look translated into `translation`, as it's three sentences.

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 4]
```diff
@@ -1771,13 +1771,17 @@ p, li { white-space: pre-wrap; }
         <source>BIP70 payment requests are deprecated and disabled by default. Restart with -enable-bip70 if you absolutely have to use this functionality.
 
 Use this functionality with extreme caution.</source>
-        <translation>해당 기능 사용시 매우 주의하기 바랍니다</translation>
+        <translation>BIP70 지불 요청은 지원 종료 되었으며 기본적으로 비활성화 되어 있습니다. 꼭 해당 기능 이용해야 한다면 -enable-bip70 이용하여 재시작하기 바랍니다.
```
Comment (ilsubyeega) [inline src/qt/locale/bitcoin_ko_KR.ts:1774]: 뒷 문장이 어색해 보입니다. 제가 암호화페에 대하여 자세히는 모르지만 -enable-bip70은 옵션/파라미터로 보여지는데 `~을 이용하여` 보단 `~와 함께`거나 `~ 옵션을 추가해` 와 같은 문구가 어울릴 것 같습니다.

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 5]
```diff
@@ -1771,13 +1771,17 @@ p, li { white-space: pre-wrap; }
...
+
+해당 기능 사용시 매우 주의하기 바랍니다</translation>
     </message>
     <message>
         <source>Payment request file handling is disabled by default. Restart with -enable-bip70 if you absolutely have to use this functionality.
 
 Use this functionality with extreme caution.</source>
-        <translation>해당 기능 사용시 매우 주의하기 바랍니다</translation>
+        <translation>Payment request file handling은 기본적으로 비활성화 되어 있습니다. 꼭 해당 기능 이용해야 한다면 -enable-bip70 이용하여 재시작하기 바랍니다.
```
Comment (ilsubyeega) [inline src/qt/locale/bitcoin_ko_KR.ts:1782]: Payment request file handling은 해당 번역 파일에 `지불 요청 파일 처리` 로 번역되어 있으니, 이점 참고하시면 좋을 것 같습ㄴ다.

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 6]
```diff
@@ -4571,11 +4575,11 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>Execute command when a wallet transaction changes (%s in cmd is replaced by TxID, %i with block height, with a value of 0 if tx is no longer in chaintip)</source>
-        <translation type="unfinished"></translation>
+        <translation>지갑 트랜잭션 변화시 명령 실행 (명령 안의 %s는 TxID, %i 는 블록높이로 변경되며 트랜잭션이 더 이상 chaintip 안이 아닌 경우에는 0으로 대체됩니다)</translation>
     </message>
     <message>
         <source>Enable BIP-70 PaymentServer (default: 0)</source>
-        <translation type="unfinished"></translation>
+        <translation>BIP-70 PaymentServer 활성화</translation>
```
Comment (ilsubyeega) [inline src/qt/locale/bitcoin_ko_KR.ts:4582]:
```suggestion
        <translation>BIP-70 PaymentServer 활성화 (기본값: 0)</translation>
```

**Review by ilsubyeega — CHANGES_REQUESTED** [review]

**Review by ilsubyeega — APPROVED** [review]

**Review by patricklodder — APPROVED** [review]
ACK - code reviewed.

Language validated and corrected by @ilsubyeega

**Pull request merged**